src/css/yeldaWebchatInjector.css

```diff
@@ -56,6 +56,10 @@
     z-index: 2147483647;
   }
 
+  .yelda_container.inner{
```

Comment:
Why is this selector two times ? can you make one that works for both @media cases ? thanks